src/actions.rs

```diff
@@ -86,6 +89,17 @@ impl<'a> Action for Step<'a> {
     async fn call(&self) -> anyhow::Result<String> {
         let gh = GithubClient::new_with_default_token(Client::new());
 
+        // retrieve all Rust compiler meetings
+        // from today for 7 days
+        let today: chrono::DateTime<chrono::Local> = chrono::Local::now();
+        let tcompiler_meetings: Vec<CompilerMeeting> = CompilerMeeting::get_meetings(
+            today.format("%F").to_string(),
+            (today + Duration::days(7)).format("%F").to_string(),
+        )
+        .await
+        .map_err(|e| eprintln!("Meetings couldn't be retrieved: {:?}", e))
```

Comment:
Do you really want to abort here?, or maybe instead of eprintln! use `vec![format!("Meetings couldn't be retrieved: {:?}", e)]`, so if for some reason an external service like google calendar is not giving you the information you need you can still run the agenda and you will see in the calendar section the text that says meetings could be retrieved.